Config/GraphQLEndpoint/Helpers/OverblogGraphQLBundleEndpointResolver.php

```diff
@@ -4,9 +4,9 @@
 
 final class OverblogGraphQLBundleEndpointResolver
 {
-    public static function getByName($name)
+    public static function getByName($name = null)
     {
-        if ('default' === $name) {
+        if ('default' === $name || null === $name) {
```

Comment:
I think we should use only `'default'` or `null`. Not both.
Depending on the decision, the default value of `$name` should be changed.